inst/include/dust/ode/dust_ode.hpp

```diff
@@ -32,7 +32,7 @@ class dust_ode {
         n_threads_(n_threads),
         shape_({n_particles}),
         m_(model_type(pars)),
-        rng_(n_particles_, seed, false),
+        rng_(n_particles_ + 1, seed, false), // +1 for filter
```

Comment:
sorry can you remind me what this is about?

Reply:
Sorry - cryptic messages in both the ticket and the description. In order to run a version of the particle filter entirely in compiled code (which gets a small speed boost) we need one extra copy of random number state, in order to draw the numbers used for reordering the particles. We made the call originally that this should not come from any of the particles themselves, so initialise an additional copy of the RNG state